cluster/test-deploy/azure/vars.yaml

```diff
@@ -0,0 +1,34 @@
+---
+openshift_azure_clusterid: magentaci
+openshift_azure_resource_location: eastus
+
+openshift_deployment_type: openshift-enterprise
+
+openshift_azure_vm_admin_username: openshift
+ansible_ssh_user: "{{ openshift_azure_vm_admin_username }}"
+ansible_ssh_private_key_file: '~/.ssh/id_rsa'
+become: true
+
+openshift_master_bootstrap_enabled: true
+openshift_master_fire_handlers: false
+openshift_master_systemd_enable: no
+openshift_is_containerized: false
+
+openshift_azure_vm_ssh_public_keys:
+- path: "/home/{{ openshift_azure_vm_admin_username }}/.ssh/authorized_keys"
```

Comment:
Why is this not consistent with the other jobs?

Reply:
Azure doesn't use cloud-init.  They use their own program to lay down users.